exporter/lokiexporter/exporter.go

```diff
@@ -32,23 +34,27 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/loki"
 )
 
-type nextLokiExporter struct {
+const (
+	maxErrMsgLen = 1024
```

Comment:
Is this used anywhere?

Reply:
yes, it is used here https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/36d526730a2453accb158dd7f4c73ce11f436993/exporter/lokiexporter/exporter.go#L112

Reply:
Oh i see it was defined in a deleted file previously. I couldn't see it in the github diff :D Thanks!